geode-core/src/main/java/org/apache/geode/distributed/internal/ReplyProcessor21.java

```diff
@@ -139,43 +140,35 @@
   protected final DM dmgr;
 
   /** Start time for replyWait stat, in nanos */
-  protected long statStart;
+  long statStart;
 
   /** Start time for ack-wait-threshold, in millis */
-  protected long initTime;
+  private long initTime;
 
   /**
    * whether this reply processor should perform severe-alert processing for the message being ack'd
    */
-  protected boolean severeAlertEnabled;
+  private boolean severeAlertEnabled;
 
   /**
    * whether the severe-alert timeout has been reset. This can happen if a member we're waiting for
    * is waiting on a suspect member, for instance.
    */
-  protected volatile boolean severeAlertTimerReset;
+  private volatile boolean severeAlertTimerReset;
 
   /**
    * whether this reply processor should shorten severe-alert processing due to another vm waiting
    * on this one. This is a thread-local so that lower level comm layers can tell that the interval
    * should be shortened
    */
-  public static final ThreadLocal SevereAlertShorten = new ThreadLocal() {
-    @Override
-    protected Object initialValue() {
-      return Boolean.FALSE;
-    }
-  };
+  private static final ThreadLocal<Boolean> SevereAlertShorten =
```

Comment:
Should `SevereAlertShorten` and `ForceSevereAlertProcessing` have a majuscule for their first letter?

Reply:
Probably not. I didn't even know that was a word.